### PR TITLE
Feature/startpage

### PR DESCRIPTION
**Pull request by frejah**
First startpage layout draft.

### CODE REVIEW

File: src/Assets/Styles/App.scss [hunk 1]
```diff
@@ -8,6 +8,8 @@ html {
 body {
   height: 100%;
   margin-bottom: 60px;
+  background-image: url('../Images/background.jpeg') !important;
```
Comment (SebastianGerS) [inline src/Assets/Styles/App.scss]: don't use !important, you can make the rule more specific instead, like so:
`html body {
  background: no-repeat url('../Images/background.jpeg');
  background-size: cover;
}`

**Review by SebastianGerS — COMMENTED** [review]
Looks good!

just don't use !important
it's important! :P

**Review by SebastianGerS — APPROVED** [review]
Added the background image back in (without !important) it works perfectly when I'm viewing it locally, but let me know if there seams to be a problem with it!

**Comment by izabelrosen** [comment]
Look at the design layout and center the buttons of the middle of the page. As of now they are at the top of the page, please center-center them. Otherwise - good work.

**Pull request merged**